pkg/printer/options.go

```diff
@@ -96,3 +90,17 @@ func IgnoreStderr() Option {
 		opts.werr = os.NewFile(uintptr(syscall.Stderr), os.DevNull)
 	}
 }
+
+// SetPortTranslation sets the flag to translate port numbers to port names, i.e. `80` becomes `80(http)`.
+func SetPortTranslation(enabled bool) Option {
```

Comment:
other options are titled `With*` or just their name, but not `Set*`

Reply:
will fix